apps/docs/docs/extended/include.model.dsl

```diff
@@ -0,0 +1 @@
+user = person "Included Person" "This DSL does not get included in the generated code as its an inclue file and not a workspace file denoted by the filename 'include.'"
```

Comment:
**praise (non-blocking)**: Thanks for adding an example! This made it very easy for me to test.